eq_translations/survey_schema.py

```diff
@@ -169,8 +169,10 @@ def get_catalog(self):
 
         for pointer in self.no_context_pointers:
             pointer_contents = resolve_pointer(self.schema, pointer)
-            if pointer_contents:
+            if isinstance(pointer_contents, str):
                 catalog.add(dumb_to_smart_quotes(pointer_contents))
+            else:
+                catalog.add(dumb_to_smart_quotes(pointer_contents.get("text")))
```

Comment:
This suggests that some pointer_contents aren't pointing to a string. Pointers should all be strings that need to be translated. Instead of updating how the catalog is read, we should update the no_context_pointers/pointers method to return the child "text" key which will mean this if statement isn't necessary.